sites/gcweb-menu/_screen-sm-max.scss

```diff
@@ -19,8 +19,6 @@
 	}
 
 	button[aria-haspopup=true] {
-		margin-left: calc( 50% - 360px ); /* 360px is half the container, minus the padding */
```

Comment:
Is this change has an impact for the previous pattern for the gcweb-menu?

I will need to test it.

Reply:
This has been tested locally and doesn't seems to have any impact. To ensure, I will do another round of testing and confirm.